src/react/components/common/customField/customFieldTemplate.tsx

```diff
@@ -2,34 +2,35 @@ import React, { Fragment } from "react";
 import { FieldTemplateProps } from "react-jsonschema-form";
 
 export default function CustomFieldTemplate(props: FieldTemplateProps) {
-    const { id, label, required, description, help, rawErrors, children } = props;
+    const { id, label, required, description, rawErrors, schema, uiSchema, children } = props;
     const classNames = [];
     if (props.schema.type === "object") {
         classNames.push("object-wrapper");
     } else {
         classNames.push("form-group");
-        if (rawErrors && rawErrors.length > 0) {
-            classNames.push("is-invalid");
-        } else {
-            classNames.push("is-valid");
-        }
+    }
+
+    if (rawErrors && rawErrors.length > 0) {
+        classNames.push("is-invalid");
+    } else {
+        classNames.push("is-valid");
     }
 
     return (
         <div className={classNames.join(" ")}>
             { /* Render label for non-objects except for when an object has defined a ui:field template */}
-            {props.schema.type !== "array" &&
-                (props.schema.type !== "object" || (props.schema.type === "object" && props.uiSchema["ui:field"])) &&
+            {schema.type !== "array" &&
+                (schema.type !== "object" || (schema.type === "object" && uiSchema["ui:field"])) &&
```

Comment:
I don't think you need to specify the schema.type === "object" condition in the OR expression